embedded-compositor/notificationmodel.h

```diff
@@ -15,8 +15,10 @@ struct Notification
     uint id = 0;
     QString summary;
     QString body;
+    QString appIcon;
     QStringList actionNames;
     QStringList actionLabels;
+    bool actionIcons;
```

Comment:
```suggestion
    bool actionIcons = false;
```
Makes the line [179](https://github.com/JUMO-GmbH-Co-KG/embedded-compositor/pull/40/files#diff-43725f3030783139059b11990ddb6f004cf8a3047f794f708a45642eb73ad9e7R179) obsolet right?